ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -177,16 +198,26 @@ tests:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-edge-zone-byo-subnet-ovn
 - as: aws-ipi-localzone-f28
-  cron: 35 9 4 * *
+  cron: 46 5 19 * *
   steps:
     cluster_profile: aws-1-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-edge-zone
-- as: aws-ipi-outpost-day2-f14
-  cron: 22 19 12,28 * *
+- as: aws-ipi-localzone-fips-f28
```

Comment:
If we add this job in amd upgrade ci, then we need to remove the one in multi-arch upgrade ci.

Reply:
Let's remove the one in multi-arch